client/components/Login.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { login } from '../reducers'
+
+export class Login extends Component {
+  constructor() {
+    super()
+    this.state = {
+      emailInput: '',
+      passwordInput: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
+  }
+
+  handleChange(evt) {
+    //HANDLE VALIDATIONS?
+    switch (evt.target.name) {
+      case 'email':
+        this.setState({emailInput: evt.target.value, dirty: true})
+        break;
+      case 'password':
+        this.setState({passwordInput: evt.target.value, dirty: true})
+        break;
+      default:
+        break;
+    }
+  }
+
+  render() {
+    const { emailInput, passwordInput} = this.state
+
+    return (
+      <div>
+        <form name="login-form" onSubmit={(evt) => this.props.handleSubmit(evt, emailInput, passwordInput)}>
+          <div>
+            <label>Email: </label>
+            <input type="text" name="email" value={emailInput} onChange={this.handleChange} required />
```

Comment:
what is the name attribute here? How is it used and/why why is it necessary?

Reply:
it is needed to grab the name of the input and the value